tests/common/gcp_type/test_data/fake_instance.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2017 Google Inc.
```

Comment:
I looked at the data in this file and I am not 100% sure it's completely scrubbed, could you confirm? Let's try to use generic ids/names (e.g. instead of "spotify-facebook-system", perhaps "project-1")